README.md

```diff
@@ -1,4 +1,4 @@
-# `chevron`: Standard TLGs for clinical trials reporting <a href='https://insightsengineering.github.io/chevron/'><img src="man/figures/chevron.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
```

Comment:
hey @llrs-roche this does not work and need a follow-up PR
https://github.com/insightsengineering/chevron?tab=readme-ov-file
![image](https://github.com/user-attachments/assets/c433c8b1-bebb-4e3e-a95a-8b15546bdb4a)

It has to be changed to `chevron.png` (or file renamed to `logo.png`)

I just picked this repo randomly. Please double check other logo updates.

Reply:
Thanks Pavel! I think I missed renaming the file to logo.svg or maybe we are stuck with png for github to show the logo as `usethis::use_logo` only uses PNG. I'll be checking this on all repos.